test/src/test/kotlin/graphql/nadel/tests/next/fixtures/defer/transforms/DeferredServiceTypeFilterTestSnapshot.kt

```diff
@@ -0,0 +1,68 @@
+// @formatter:off
+package graphql.nadel.tests.next.fixtures.defer.transforms
+
+import graphql.nadel.tests.next.ExpectedNadelResult
+import graphql.nadel.tests.next.ExpectedServiceCall
+import graphql.nadel.tests.next.TestSnapshot
+import graphql.nadel.tests.next.listOfJsonStrings
+import kotlin.Suppress
+import kotlin.collections.List
+import kotlin.collections.listOf
+
+private suspend fun main() {
+    graphql.nadel.tests.next.update<DeferredServiceTypeFilterTest>()
+}
+
+/**
+ * This class is generated. Do NOT modify.
+ *
+ * Refer to [graphql.nadel.tests.next.UpdateTestSnapshots
+ */
+@Suppress("unused")
+public class DeferredServiceTypeFilterTestSnapshot : TestSnapshot() {
+    override val calls: List<ExpectedServiceCall> = listOf(
+            ExpectedServiceCall(
+                service = "A",
+                query = """
+                | {
+                |   aErrors {
+                |     __typename__type_filter__id: __typename
+                |   }
+                | }
+                """.trimMargin(),
+                variables = " {}",
+                result = """
+                | {
+                |   "data": {
+                |     "aErrors": {
+                |       "__typename__type_filter__id": "AError"
+                |     }
+                |   }
+                | }
+                """.trimMargin(),
+                delayedResults = listOfJsonStrings(
+                ),
+            ),
+        )
+
+    /**
+     * ```json
+     * {
+     *   "data": {
+     *     "aErrors": {}
+     *   }
+     * }
+     * ```
+     */
+    override val result: ExpectedNadelResult = ExpectedNadelResult(
+            result = """
+            | {
+            |   "data": {
+            |     "aErrors": {}
```

Comment:
There's not incremental data in the assertion here as well.

Reply:
In this test, there is not supposed to be incremental data, because there is no data in the **aErrors** field.

This test is to ensure that the **BError** type does not get returned in the **aErrors** field